# Trajectory Message for Landing Planner
# Contains polynomial coefficients for differential flatness trajectory

Header header

float64[8] x_coeffs
float64[8] y_coeffs
float64[8] z_coeffs

float64[4] yaw_coeffs

# Trajectory duration (seconds)
float64 duration
